src/app.ts

```diff
@@ -34,6 +35,16 @@ export function buildApp(
 	getPanda: () => PanDomainAuthentication = getPanDomainAuth,
 ): Express {
 	const app = express();
+
+	const allowedOrigins = [/gutools\.co\.uk$/];
```

Comment:
I _think_ this would match `mydodgydomaingutools.co.uk`. Would it be worth changing this to `/\.gutools\.co\.uk$/`? We'd probably then need to add `/\.dev-gutools\.co\.uk$/ to the list too. (Or could possibly combine into `/\.(dev-)?gutools.co.uk$/`).